README.md

```diff
@@ -222,7 +222,7 @@ PcapPlusPlus currently supports parsing, editing and creation of packets of the
 33. HTTP headers (request & response)
 34. NTP (v3, v4)
 35. Radius
-36. SOME/IP
+36. SOME/IP (SOME/IP, SOME/IP-TP)

```

Comment:
I'm not sure if we should provide this extra info. Soon we'll merge the SD PR and then we'll cover all SOME/IP packets, right?